lib/curlx.h

```diff
@@ -112,6 +112,7 @@
 # define mvsnprintf curlx_mvsnprintf
 # define aprintf curlx_maprintf
 # define vaprintf curlx_mvaprintf
+# define vsprintf curlx_mvsprintf
```

Comment:
No need to add this define. This function should not be used in libcurl code.

Reply:
OK, then `# undef vsprintf` can be safely deleted few lines above, isn't it?

Reply:
Yes, it seems to serve no purpose.